api/report/undoc/b_api.c

```diff
@@ -937,6 +1008,36 @@ int IodeModelSimulate(char *per_from, char *per_to, char *eqs_list, char *endo_e
 }
 
 
+double IodeModelSimNorm(char* period)
+{
+    U_ch *args[2];
+    
+    args[0] = period;
+    args[1] = 0;
+    
+    return(RPF_SimNormReal(args));
+}
+
+int IodeModelSimNIter(char* period)
+{
+    U_ch *args[2];
+    
+    args[0] = period;
+    args[1] = 0;
+    return(RPF_SimNIterInt(args));
+}
+    
+int IodeModelSimCpu(char* period)
```

Comment:
Could you check if there is a corresponding C++ method in the Simulation class (see file `cpp_api\compute\simulation.h`) ? 
If not, could add a TODO comment in the `cpp_api\compute\simulation.h` file ? 
Thanks